iiod/ops.h

```diff
@@ -19,31 +19,31 @@
 #ifndef __OPS_H__
 #define __OPS_H__
 
-#include "../iio-private.h"
-#include "queue.h"
-
 #include <endian.h>
 #include <errno.h>
+#include <poll.h>
 #include <stdint.h>
 #include <stdio.h>
 #include <string.h>
-#include <poll.h>
 #include <sys/socket.h>
 #include <unistd.h>
 
+#include "../iio-private.h"
+#include "queue.h"
+
 #if WITH_AIO
 #include <libaio.h>
 #endif
 
 #ifndef __bswap_constant_16
-#define __bswap_constant_16(x) \
-	((unsigned short int) ((((x) >> 8) & 0xff) | (((x) & 0xff) << 8)))
+#define __bswap_constant_16(x)                                                 \
+	((unsigned short int)((((x) >> 8) & 0xff) | (((x)&0xff) << 8)))
 #endif
 
 #ifndef __bswap_constant_32
-#define __bswap_constant_32(x) \
-	((((x) & 0xff000000) >> 24) | (((x) & 0x00ff0000) >>  8) | \
-	 (((x) & 0x0000ff00) <<  8) | (((x) & 0x000000ff) << 24))
+#define __bswap_constant_32(x)                                                 \
+	((((x)&0xff000000) >> 24) | (((x)&0x00ff0000) >> 8) |                  \
```

Comment:
this is weird; it removes spaces around bitwise op

Reply:
Yeah this weird - maybe a bug in clang-format?

Reply:
SpacesInParentheses is set to false.